examples/lighting-app/nrfconnect/prj.conf

```diff
@@ -128,6 +128,12 @@ CONFIG_OPENTHREAD_ENABLE_SERVICE=y
 CONFIG_OPENTHREAD_JOINER=y
 CONFIG_OPENTHREAD_MANUAL_START=y
 
+# Default OpenThread network settings
+CONFIG_OPENTHREAD_PANID=4660
+CONFIG_OPENTHREAD_CHANNEL=15
+CONFIG_OPENTHREAD_NETWORK_NAME="OpenThread"
```

Comment:
Wasn't this `OpenTreadDemo` ?

Reply:
It was but in fact it doesn't matter as CHIP node always gets name from the leader, so in case of using OTBR it will override this name anyway.